third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -935,6 +935,10 @@ static void SHA256DigestHex(const v8::FunctionCallbackInfo<v8::Value>& args) {
     sprintf(digestHex + i * 2, "%02x", digest[i]);
   }
 
+  // The content being hashed can vary when replaying if source contents have been
+  // replaced, so we record/replay the result.
+  recordreplay::RecordReplayBytes("SHA256DigestHex", digestHex, 65);
```

Comment:
When replaying, why do we even do the hashing work if we already have the result?

Reply:
I don't know what you mean, this is code that runs both when recording and replaying and our focus is making sure it behaves consistently.  This leads to different filenames being computed and accessed, and while I'm not sure if that will cause immediate problems (since the access results are replayed from the recording it doesn't matter so much if the filename is different) it can mask other later issues.